querybot/static/js/script.js

```diff
@@ -209,24 +226,50 @@ async function executeQuery() {
           <h6>SQL Query Execution Result:</h6>
           <div id="sqlResultTable" class="table-responsive" style="max-height: 50vh;"></div>
           <div class="mt-3">
-            <button class="btn btn-primary me-2" @click=${() => downloadCSV(result.result, "query_result.csv")}>
-              <i class="bi bi-download"></i> Download Results as CSV
-            </button>
-            <div class="row mt-2">
-              <div class="col-md-8">
+            <div class="row align-items-center g-2">
+              <div class="col-auto">
+                <button class="btn btn-primary me-2" @click=${() => downloadCSV(result.result, "query_result.csv")}>
+                  <i class="bi bi-download"></i> Download Results as CSV
+                </button>
+              </div>
+              <div class="col">
                 <input
                   type="text"
-                  id="additionalPrompt"
+                  id="chart-input"
                   class="form-control"
-                  placeholder="Optional: Add specific instructions for the explanation..."
+                  placeholder="Describe what you want to chart"
+                  value="Draw the most appropriate chart to visualize this data"
                 />
               </div>
-              <div class="col-md-4">
-                <button class="btn btn-info" @click=${() => explainResults(result.result, query)}>
-                  <i class="bi bi-lightbulb"></i> Explain Results
+              <div class="col-auto">
```

Comment:
If we use col earlier, might be best to stick to col (not col-auto) here as well